client/src/app/primitives/modal/Modal.js

```diff
@@ -62,10 +60,12 @@ export default class Modal extends PureComponent {
     } = this.props;
 
     return ReactDOM.createPortal(
-      <div className={ css.ModalOverlay }>
-        <div className={ classNames(css.ModalContainer, className) } ref={ this.modalRef }>
-          { children }
-          { onClose && (<Close onClick={ this.close } />) }
+      <div className="modal" tabIndex="-1" role="dialog">
```

Comment:
Why do we need `tabIndex` here? AFAIK `div`s are not focusable per default.

Reply:
This should answer it: https://stackoverflow.com/questions/32911355/whats-the-tabindex-1-in-bootstrap-for

TLDR: With `tabIndex=-1` we can focus the element programmatically. We do not do that yet but _could_ do it. It also helps a11y technologies.